src/Compilers/CSharp/Portable/Syntax/LookupPosition.cs

```diff
@@ -228,6 +228,36 @@ internal static bool IsInMethodTypeParameterScope(int position, MethodDeclaratio
             return !IsBetweenTokens(position, firstNameToken, firstPostNameToken);
         }
 
+        internal static bool IsInLocalFunctionTypeParameterScope(int position, LocalFunctionStatementSyntax localFunction)
+        {
+            Debug.Assert(localFunction != null);
+
+            if (localFunction.TypeParameterList == null)
+            {
+                // no type parameters => nothing can be in their scope
```

Comment:
>nothing can be in their scope [](http://example.com/codeflow?start=41&length=29)

The wording looks strange. Type parameters can be in scope rather than something else can be in their scope.